documentation/Database-Schema-Permission.md

```diff
@@ -5,30 +5,45 @@
 ## Database Area description: Permission
 
 ### Definition
+
 To define permissions with actionables for the user over an application templates.
 
 We like to define the permissions definition using: Who, how and what.
-Who => Is the `permission_name`, which describes the type of action to be associated with users.
-How => Is the `permission_policy` or how the user that has this permission can interact with an application template (through actions).
-What => Is the `template_permission` that connects some permission with an application template or section of an application template.
 
-Before any user is included, permissions can be created. The permission definition consists only on the `permission_policy` and the `permission_name`. And then what links this permission with an application template is the `template_permission`, and after the permission is granted to a user (or a user in a company) what links this permission with an user is the `permission_join`.
+**Who** => Is the `permission_name`, which describes the type of action to be associated with users.
+
+**How** => Is the `permission_policy` or how the user that has this permission can interact with an application template (through actions).
```

Comment:
Also, when you say "actions" here, you don't mean "Actions", (as in Action plugins), right? Perhaps "operations" would be better to avoid confusing terminology?

Reply:
Oh yea! Good point. I'll edit it now.